### DISCUSSION TITLE
Specifying POST Request

### DISCUSSION THREAD

**Discussion by musicjoeyoung** [mockapi-io/docs#12] [Q&A]
Is it possible to edit a POST request so that I can I add data into individual users? As an example, I was hoping to provide the option--in a practice project--to add additional comments to a user. I'm not sure if this is a possibility. Any thoughts are greatly appreciated. 

Here is my JSON structure:
`{
  "id": "3",
  "firstName": "$name.firstName",
  "lastName": "$name.lastName",
  "job": "$name.jobTitle",
  "image": "$internet.avatar",
  "bio": "$lorem.sentence",
  "comments": [
    {
      "id": "$datatype.uuid",
      "comment": "$random.words"
    },
    {
      "id": "$datatype.uuid",
      "comment": "$random.words"
    },
    {
      "id": "$datatype.uuid",
      "comment": "$random.words"
    }
  ]
}`

**Comment by nsmirn0v** [comment]
This is possible. Just add whatever data you need to the POST body and mockAPI will handle it. It doesn't have to match resource schema.
